docs/en/observability/observability-ui.asciidoc

```diff
@@ -27,42 +27,41 @@ Additionally, the alerts chart keeps you informed of any issues that you may nee
 to resolve quickly.
 
 [float]
-[[view-services-and-transactions]]
-== Services and transactions
-
-The *APM* chart displays the total number of services running within your environment
-and the total number of transactions per minute that were captured by the Elastic APM
-agent instrumenting those services.
+[[view-alerts]]
+== Alerts
 
-To drill-down and view this APM data in the *APM app*, click *View in app*.
+To help keep you aware of potential issues in your environments, the *Alerts* feed
```

Comment:
```suggestion
To help keep you aware of potential issues in your environments, the *Alerts* view
```
Is it a feed or a view (or something else)?

Reply:
Maybe "table"?